### PR TITLE
Final 1.6 copy for overall plugin UI

### LINKED ISSUE #4625: Update Reader Mode Settings message
In an upcoming version (e.g. 1.6) the description of the Reader mode in the Setting page should be updated to define the clear purpose and value of the mode, and remove a reference to the classic mode.

### LINKED ISSUE #4684: Improve Copy of AMP Settings page
- The wording of the Settings page is loaded with technical jargon that could be rephrased in a more accessible and clear way.
- The structure of the page should be revised

/cc @aarondruck @jwold

 
review-settings-page-copy

enhace-settings-page-2

### LINKED ISSUE #4871: Setup wizard: Final copy for all screens
## Feature description

Creating this issue to track the creation of copy for the setup wizard. Having this handled separately will allow us to avoid having design and engineering held up by wording concerns until the finished product is ready. At that time we can put in all the hard-coded text in one sitting.

## Acceptance criteria

* Clear, translatable, user-friendly text on all screens

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

### PR DESCRIPTION
**Pull request by johnwatkins0**
[WIP: depends on #4998 and will require feedback from everyone]

## Summary

Fixes #4684 
Fixes #4871 
Fixes #4625 

## Checklist

- [ ] My pull request is addressing an open issue (please create one otherwise).
- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [ ] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: assets/src/onboarding-wizard/pages/save/index.js [hunk 1]
```diff
@@ -210,7 +210,7 @@ export function Save() {
 			<div className="done__preview-container">
 				{ 'reader' === themeSupport && downloadingThemeError && (
 					<AMPNotice size={ NOTICE_SIZE_LARGE } type={ NOTICE_TYPE_WARNING }>
-						{ __( 'There was an error downloading your reader theme. As a result, your site is currently using the legacy reader theme. Please install your chosen theme manually.', 'amp' ) }
+						{ __( 'There was an error downloading your reader theme. As a result, your site is currently using the legacy reader theme. Please install your chosen theme by manually.', 'amp' ) }
```
Comment (pierlon) [inline assets/src/onboarding-wizard/pages/save/index.js]: > ...install your chosen theme by manually.

Typo here?

Comment (johnwatkins0) [reply assets/src/onboarding-wizard/pages/save/index.js]: Yup. Fixed in 599b5396a

**Comment by amedina** [comment]
Looking good. We will do an additional pass over the copy for all sections
of the plugin once the PRs are done.

**Comment by pierlon** [comment]
In the welcome section on the settings page, the wizard is referred to as "configuration", but in the admin menu (and about every where else) it's referred to as "onboarding".


image

**Comment by johnwatkins0** [comment]
> In the welcome section on the settings page, the wizard is referred to as "configuration", but in the admin menu (and about every where else) it's referred to as "onboarding".

I personally haven't felt great about the word "onboarding" because "onboarding" to me refers to the stuff I do when I start a new job. I also didn't think "setup" sounded right (hence as you saw in #4998 @pierlon I removed `setup` in favor of `onboarding`). But I think "Configuration Wizard" might be a better term. Of course changing this would end up creating another large diff as I search and replace everything again, but once it's settled we'll be able to leave it alone for good.

**Comment by johnwatkins0** [comment]
Also noting, as Pierre brought up in https://github.com/ampproject/amp-wp/pull/4998/files#r451950271, we should establish consistency in wither "Standard," "Transitional," and "Reader" are capitalized (e.g., "Your site is using Standard mode" vs "Your site is using standard mode")

**Comment by westonruter** [comment]
Closing per https://github.com/ampproject/amp-wp/issues/4625#issuecomment-669483543

**Pull request closed without merging**